CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Refactored Postman CLI files to make them work from Postman folder. - [#304](https://github.com/DigitalExcellence/dex-backend/issues/304)
```

Comment:
I'm not sure if this need an entry in the changelog

Reply:
It's fine. Rule of thumb: Everything needs to be in the changelog unless explicitly told to not do so by me project lead(s). 
I update changelogs before releases and make sure everything is correct.